tests/v2/validation/rbac/rbac_additional_test.go

```diff
@@ -60,116 +52,131 @@ func (rb *RBACAdditionalTestSuite) SetupSuite() {
 	require.NoError(rb.T(), err)
 }
 
-func (rb *RBACAdditionalTestSuite) ValidateAddStdUserAsProjectOwner() {
+func (rb *RBACAdditionalTestSuite) TestRbacP1ClusterOwner() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
 
-	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+	clusterAdmin, standardUserClient, err := setupUser(rb.client, standardUser)
 	require.NoError(rb.T(), err)
-	rb.standardUserCOProject = createProjectAsCO
 
-	log.Info("Validating if cluster owner can add a user as project owner in a project")
-	err = users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleProjectOwner, nil)
+	rb.T().Logf("Creating an additional standard user")
+	additionalUser, additionalUserClient, err := setupUser(rb.client, standardUser)
 	require.NoError(rb.T(), err)
-	userGetProject, err := projects.GetProjectList(rb.additionalUserClient, rb.cluster.ID)
+
+	rb.T().Logf("Adding user as " + roleOwner + " to the downstream cluster.")
```

Comment:
is roleOwner defined somewhere?

Reply:
yes in the rbac.go file. this is a constant as well.